docs/advanced.rst

```diff
@@ -585,6 +585,106 @@ Discovery
 .. END INTRODUCTION
 
 .. START SIMPLE DISCOVERY
+
+
+SIMPLE Discovery Settings
+=========================
+
+The SIMPLE discovery protocol resolves the establishment of the end-to-end connection between various
+RTPS entities communicating via the RTPS protocol. Fast-RTPS implements the SIMPLE discovery protocol to
+provide compatibility with the `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_.
+The specification splits up the SIMPLE discovery protocol into two independent protocols:
+
+- **Simple Participant Discovery Protocol (SPDP):** specifies how Participants discover each other in the network; it
+  announces and detects the presence of participants in a domain.
+
+- **Simple Endpoint Discovery Protocol (SEDP):** defines the protocol adopted by the discovered participants for the
+  exchange of information in order to discover the RTPS entities contained in each of them, i.e. the writers and
+  readers Endpoints.
+
```

Comment:
I would add a sentence here stating that the following table describes the possible settings for SIMPLE discovery